sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -19,6 +19,8 @@ package org.apache.spark.sql.sources
 
 import java.util.{Date, UUID}
 
+import scala.collection.JavaConversions._
```

Comment:
I explicitly didn't include this wildcard implicit import because I didn't want future code to accidentally introduce a scala wrapper on the java hashmap.